html/pfappserver/root/static.alt/src/components/pfFormSortableFields.vue

```diff
@@ -77,7 +77,8 @@
 -->
 <template>
   <b-form-group horizontal :label-cols="(columnLabel) ? labelCols : 0" :label="$t(columnLabel)"
-    :state="isValid()" :invalid-feedback="getInvalidFeedback()" :class="['sortablefields-element', { 'is-focus': drag }, { 'mb-0': !columnLabel }]"
+    :state="isValid()" :invalid-feedback="getInvalidFeedback()" 
+    class="sortablefields-element" :class="[{ 'is-focus': drag }, { 'mb-0': !columnLabel }]"
```

Comment:
You can simplify to a single object literal: `:class="{ 'is-focus': drag, 'mb-0': !columnLabel }"`